### PR TITLE
fix(pkg): fix gcc version from builder images setter.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

> /area pkg

**What this PR does / why we need it**:

Skip all the fallback logic if we find an image providing the correct gcc in the first place.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/driverbuilder/builder/builders.go [hunk 1]
```diff
@@ -185,31 +185,31 @@ func (b *Build) setGCCVersion(builder Builder, kr kernelrelease.KernelRelease) {
...
-	// that is, the list of available gccs from images
-	// for each builder image
-	proposedGCCs := make([]semver.Version, 0)
-	for _, img := range b.Images {
-		proposedGCCs = append(proposedGCCs, img.GCCVersion)
-		logger.WithField("image", img.Name).
-			WithField("targetGCC", targetGCC.String()).
-			Debug("proposedGCC=", img.GCCVersion.String())
-	}
+	} else {
```
Comment (FedeDP) [inline pkg/driverbuilder/builder/builders.go:188]: If we found a `gccversion` provided by an image, there is no need to fallback at secondary algorithm.

**Comment by EXONER4TED** [comment]
/approve

One more try 😅

**Comment by EXONER4TED** [comment]
Nope, it wants someone other than me 🤷 Looks good to me though :)

**Review by EXONER4TED — APPROVED** [review]

**Pull request merged**